fire/cli/ts/_plot_gnss.py

```diff
@@ -0,0 +1,283 @@
+    plottes hvis flaget ``ErPooled`` sættes.
+
+    ``alpha`` bestemmer signifkansniveauet for konfidensbånd til fittet samt for hypo-
+    tesetests.
+
+    Reference-hældningen vises som en linje der skærer regressionslinjen i punktet (mex,
+    mey), som er punktet i midten af tidsseriens tidsmæssige spænd (middelepoken).
+    """
+
+    # Prædiktioner og intervaller
+    x_præd = np.linspace(linreg.x[0], linreg.x[-1], 1000)
+    y_præd = linreg.beregn_prædiktioner(x_præd)
+
+    konfidensbånd = linreg.beregn_konfidensbånd(
+        x_præd,
+        y_præd,
+        alpha=alpha,
+        ErPooled=False,
+    )
+
+    konfidensbånd_pooled = linreg.beregn_konfidensbånd(
+        x_præd,
+        y_præd,
+        alpha=alpha,
+        ErPooled=True,
+    )
+
+    # Uplift
+    uplift_rate = linreg.hældning_reference
+    uplift_fit = uplift_rate * (x_præd - linreg.mex) + linreg.mey
+
+    # Hypotesetest
+    H0 = uplift_rate - linreg.beta[1]
+
+    T_test = linreg.beregn_hypotesetest(H0=H0, alpha=alpha)
+    Z_test = linreg.beregn_hypotesetest(H0=H0, alpha=alpha, ErPooled=True)
+
+    # Plotting
+    plt.rcParams["figure.autolayout"] = True
+    plt.figure(figsize=(12, 9))
+    ax = plt.subplot(111)
+    ax.scatter(linreg.x, linreg.y, marker="*", color="black", label="GNSS Observation")
+
+    ax.plot(
+        x_præd,
+        y_præd,
+        "r",
+        label=f"Hældning af fit: {linreg.beta[1]:.3f} [mm/år]",
+    )
+    ax.plot(
+        x_præd,
+        uplift_fit,
+        "k",
+        label=f"Uplift-rate: {uplift_rate:.3f} [mm/år]",
+    )
+
+    # Konfidensbånd
+    ax.plot(x_præd, konfidensbånd[0, :], color="green")
+    ax.plot(x_præd, konfidensbånd[1, :], color="green", label="95% Konfidensbånd")
+
+    # Konfidensbånd (pooled)
+    if ErPooled:
+        ax.plot(
+            x_præd,
+            konfidensbånd_pooled[0, :],
+            color="blue",
+        )
+        ax.plot(
+            x_præd,
+            konfidensbånd_pooled[1, :],
+            color="blue",
+            label="95% Konfidensbånd (pooled)",
+        )
+
+    ax.scatter(
+        linreg.mex,
+        linreg.mey,
+        marker="o",
+        facecolors="none",
+        edgecolors="blue",
+        label="Middelepoke",
+    )
+
+    ax.set_title(
+        f"Tidsserie: {linreg.tidsserie.navn}    R$^2$ = {linreg.R2:.2f}   N = {len(linreg.tidsserie.decimalår)}   N$\\mathregular{{_{{binned}}}}$ = {linreg.N}"
+    )
+    ax.set_xlabel("Dato")
+    ax.set_ylabel(label)
+
+    ax.yaxis.set_major_formatter(FormatStrFormatter("%.3f"))
+
+    ax.grid()
+    ax.legend(
+        loc="upper center",
+        bbox_to_anchor=(0.2, -0.15),
+        fancybox=True,
+        shadow=True,
+        ncol=1,
+    )
+
+    # T-test resultater til visning
+    t_tekst = f"|t| = {T_test.score:.2f}\nt$\\mathregular{{_{{crit}}}}$ = {T_test.kritiskværdi:.2f}\n"
+
+    if T_test.H0accepteret:
+        t_tekst += f"H$_{{0}}$ accepteret ved {T_test.alpha*100}% signifikansniveau"
```

Comment:
Formattering kan overlades til Python... Men hvis antal betydende cifre er stort, vil tallene bliver trunkeret. Du bestemmer:)

```suggestion
        t_tekst += f"H$_{{0}}$ accepteret ved {T_test.alpha:.2%} signifikansniveau"
```